### ISSUE TITLE
EssentialsX cannot skip the night

### ISSUE THREAD

**Issue by yasakamagatama1** [EssentialsX/Essentials#4235]
### Type of bug

Compatibility issue, Other unexpected behaviour

### `/ess version` output

Server version: 1.14.3-R0.1-SNAPSHOT git-Spigot-935adb3-a0e4747 (MC: 1.14.3)
EssentialsX version: 2.17.0.0
PermissionsEx version: 1.23.4
Vault version: 1.7.2-b107
EssentialsXSpawn version: 2.17.0.0
EssentialsXChat version: 2.17.0.0

### Server startup log

https://mclo.gs/qUjvI1C

### EssentialsX config files

Default config

### Bug description

When I sleep, the night did not skip at all.

### Steps to reproduce

1. I sleep
2. I cannot skip the night
3. The night still goes on so i must use scripts

### Expected behaviour

I want the night to be skipped to day

### Actual behaviour

The night did not skip

**Comment by NamelessFTW** [comment]
Hi,
I have this issue on a later version of EssentialsX on Spigot for 1.17. I believe this issue is caused by an issue in Spigot for 1.17, which I have submitted a bug report for. This should hopefully get fixed soon, but I'm not sure how often Spigot merges changes into their code.

In the meantime, a workaround for this is to remove the `essentials.sleepingignored` permission from players; this should allow them to sleep properly. I've tested that this works with a single player logged on. I'm not sure exactly what happens with multiple players online, but it should work fine if they all have the permission disabled. I don't know what happens if some players have it enabled and some don't, but it should work properly.
